JenkinsfileProductionRelease

```diff
@@ -15,8 +15,8 @@ node('linux && docker') {
     
     stage('Npm publish') {
       withCredentials([
-      string(credentialsId: 'npmjs_com_token', variable: 'NPM_TOKEN')]) {
-        sh "echo //registry.npmjs.org/:_authToken=${NPM_TOKEN} > ~/.npmrc"
```

Comment:
Jenkins gave some warnings about using string interpolation with double quotes.

According to [this doc](https://www.jenkins.io/doc/pipeline/steps/credentials-binding/), it's better to use single quotes.

Reply:
Hmm, are you sure we can remove the braces? I did not see an example without them,